File: Daily Challanges/2026-02-February/2026-02-19.js
function avalancheRisk(snowDepth, slope) {
  if(snowDepth === "Shallow" || slope ==="Gentle"){
    return "Safe"
  }
  return "Risky";
}

const test1 = avalancheRisk("Shallow", "Gentle");
const test2 = avalancheRisk("Shallow", "Steep");
const test3 = avalancheRisk("Shallow", "Very Steep");
const test4 = avalancheRisk("Moderate", "Gentle");
const test5 = avalancheRisk("Moderate", "Steep");
const test6 = avalancheRisk("Moderate", "Very Steep");
const test7 = avalancheRisk("Deep", "Gentle");
const test8 = avalancheRisk("Deep", "Steep");
const test9 = avalancheRisk("Deep", "Very Steep");

const check1 = "Safe";
const check2 = "Safe";
const check3 = "Safe";
const check4 = "Safe";
const check5 = "Risky";
const check6 = "Risky";
const check7 = "Safe";
const check8 = "Risky";
const check9 = "Risky";

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);
console.log(`Test6: ${test6 === check6}`);
console.log(`Test7: ${test7 === check7}`);
console.log(`Test8: ${test8 === check8}`);
console.log(`Test9: ${test9 === check9}`);

File: Daily Challanges/2026-02-February/2026-02-19.py
def avalanche_risk(snow_depth, slope):
    if snow_depth == "Shallow" or slope == "Gentle":
        return "Safe"
    return "Risky"

test1 = avalanche_risk("Shallow", "Gentle")
test2 = avalanche_risk("Shallow", "Steep")
test3 = avalanche_risk("Shallow", "Very Steep")
test4 = avalanche_risk("Moderate", "Gentle")
test5 = avalanche_risk("Moderate", "Steep")
test6 = avalanche_risk("Moderate", "Very Steep")
test7 = avalanche_risk("Deep", "Gentle")
test8 = avalanche_risk("Deep", "Steep")
test9 = avalanche_risk("Deep", "Very Steep")

check1 = "Safe"
check2 = "Safe"
check3 = "Safe"
check4 = "Safe"
check5 = "Risky"
check6 = "Risky"
check7 = "Safe"
check8 = "Risky"
check9 = "Risky"

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")
print(f"Test6: {test6 == check6}")
print(f"Test7: {test7 == check7}")
print(f"Test8: {test8 == check8}")
print(f"Test9: {test9 == check9}")